### ISSUE TITLE
Splits are hard to see

### ISSUE THREAD

**Issue by jonascarpay** [dylanaraps/wal.vim#19]
Similar to #13.
It can be hard to see splits, as shown in the picture below. Is there an easy fix for this?

![2019-11-24_15-02](https://user-images.githubusercontent.com/3593851/69490448-9211a000-0ecb-11ea-9312-4edcb1eb4fab.png)

**Comment by bradwood** [comment]
Having exactly the same issue? Any way to fix?

**Comment by kyokeunpark** [comment]
I made a PR for it, but essentially two attributes that you should look for is "StatusLine" and "StatusLineNC".

**Comment by bradwood** [comment]
HI @kyokeunpark -- thanks for your work on this. Unfortunately, for me, it has no effect. I use https://github.com/itchyny/lightline.vim -- it may be conflicting with this :/

**Comment by kyokeunpark** [comment]
Hi, it looks like your issue is just dependent on the lightline theme. I played around with the lightline theme for pywal and [here's what I got](https://gist.github.com/kyokeunpark/ec414efc8466e56b86cb753b58939dec). 

It looks like this to me:
![1586800427_04_13](https://user-images.githubusercontent.com/24630595/79145405-96f9c200-7d8e-11ea-85b5-0c33a1f0968d.png)

Let me know if it works for you as well.

**Comment by bradwood** [comment]
Thanks again @kyokeunpark..

I get this

![image](https://user-images.githubusercontent.com/3472045/79261674-17501e00-7e88-11ea-89e3-c565880f761f.png)

As a side effect, it doesn't seem to follow all of `wal`'s colors either, but it _does_ show splits a lot better...

**Comment by kyokeunpark** [comment]
Ah! Could you try using the wal.vim file from #26 ?

**Comment by bradwood** [comment]
I get the same error message... THis file looks a bit weird though: 

is it this one: https://raw.githubusercontent.com/dylanaraps/wal.vim/f86eaff55b9cb8d618f9575f75402e16b75fc8ae/colors/wal.vim

**Comment by kyokeunpark** [comment]
Yes, that does seem to be the one I was referring to. But, I should clarify that that's the vim colorscheme, not lightline. Sorry, I should've clarified that.
